docker/Dockerfile-dev

```diff
@@ -1,6 +1,6 @@
 FROM node:16.13.2-alpine as builder
 
-WORKDIR /usr/src/love/love
+WORKDIR /usr/src/love
```

Comment:
I was checking this and I think this should remain as `/usr/src/love/love`. This is because of the LOVE-frontend structure:
LOVE-frontend > love > source files. As you can see on https://github.com/lsst-ts/LOVE-integration-tools/blob/develop/deploy/local/live-csc/docker-compose.yml#L417 we mount the LOVE-frontend folder on `/usr/src/love`, so to access for example to the `package.json` file we have to set WORKDIR as `/usr/src/love/love`

Reply:
Checked https://github.com/lsst-ts/LOVE-integration-tools/blob/u/mareuter/dev-build/deploy/local/build/docker-compose.yml#L194. I think we can use that configuration.